src/plugins/code.spec.js

```diff
@@ -29,12 +29,12 @@ describe('Code Plugin', () => {
     `);
     const path = 'foo.html';
     const result = plugin.preload(content, path);
-    expect(result.toString()).toContain('&lt;p>My content&lt;/p>');
+    expect(result.toString()).toContain('&lt;p&gt;My content&lt;/p&gt;');
```

Comment:
It seems like node is automagically replacing brackets with HTML entities when we pull the content from the code block. 
I'm going to pull this in to one of my consuming libraries and ensure it all still works as expected.